src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -548,9 +545,51 @@ private void SetDirty(TimeSpan? maxDelay)
             return applicationData;
         }
 
+        private Dictionary<string, ConfigurationSetting> ReorderData(Dictionary<KeyValueIdentifier, string> cachedInfoLogs = default)
```

Comment:
`cachedInfoLogs` I'm not seeing where logging is coming into the picture here.

Reply:
Related to other comment about removed log from line 387.